plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -205,6 +205,19 @@ func buildControllerRoles() ([]rbacv1.ClusterRole, []rbacv1.ClusterRoleBinding)
 		},
 	})
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.DynamicResourceAllocation) {
+		addControllerRole(&controllerRoles, &controllerRoleBindings, rbacv1.ClusterRole{
+			ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + "resource-claim-controller"},
+			Rules: []rbacv1.PolicyRule{
+				rbacv1helpers.NewRule("get", "list", "watch").Groups(legacyGroup).Resources("pods").RuleOrDie(),
+				rbacv1helpers.NewRule("update").Groups(legacyGroup).Resources("pods/finalizers").RuleOrDie(),
+				rbacv1helpers.NewRule("get", "list", "watch", "create").Groups(resourceGroup).Resources("resourceclaims", "resourceclaimtemplates").RuleOrDie(),
```

Comment:
Why do we need to create resourceclaimtemplates ?

Reply:
My understanding is that @alculquicondor and @pohly made the decision during PR review to pull the `ResourceClaimTemplate` out into it's own user-facing object and add it to the `resource.k8s.io/v1alpha` API group.

As such it is no longer embedded in the Pod spec, but rather referenced by name so needs to be accessible by the plugins in order to extract any needed information from them.

I'm personally not a huge fan of losing the ability to embed the template (because it makes end user's code much more verbose even in the simplest cases of using `ResourceClaims`), but if that's what's necessary for getting this out the door as `alpha`, I'm OK with it.

Reply:
Urgh. We don't. Removed.